btree/btree.c

```diff
@@ -3992,7 +4121,7 @@ static int64_t btree_del_resolve_underflow(struct m0_btree_op *bop)
 	struct nd              *root_child;
 
 	do {
-		lev->l_freenode = true;
+		lev->l_node->n_free = true;
```

Comment:
The *_tick() functions cannot access the variables contained in the node structure. Also please rethink of changing this code as the concept of  'n_free' or 'delayed-free' is an internal node implementation detail which should not be known to the *_tick() functions.

Reply:
done